src/cloudai/cli/handlers.py

```diff
@@ -273,10 +288,22 @@ def handle_verify_all_configs(args: argparse.Namespace) -> int:
 
 
 def load_tomls_by_type(tomls: List[Path]) -> dict[str, List[Path]]:
-    files: dict[str, List[Path]] = {"system": [], "test": [], "scenario": [], "unknown": []}
+    files: dict[str, List[Path]] = {
+        "system": [],
+        "test": [],
+        "scenario": [],
+        "hook_test": [],
+        "hook": [],
+        "unknown": [],
+    }
     for toml_file in tomls:
         content = toml_file.read_text()
-        if "scheduler =" in content:
+        if "conf" in toml_file.parts and "hook" in toml_file.parts:
```

Comment:
Still, would be better to check if a TOML is under HOOK_ROOT, pathlib has something like relative_to, maybe it can be useful here.

Reply:
Updated.